src/dotfile.ts

```diff
@@ -106,7 +106,7 @@ export const DOTFILE = {
         const buffer = stripBom(fs.readFileSync(DOT.IGNORE.PATH, FS_OPTIONS));
         resolve(splitLines(buffer).filter((name: string) => name));
       } else {
-        resolve(['**/**', '!appsscript.json', '!*.js', '!*.ts']);
+        resolve(['**/**', '!appsscript.json', '!*.js', '!*.ts', '!*.html']);
```

Comment:
We have an object of all valid file extensions. This includes .gs

Can we use that object with constants instead of this?

Reply:
@grant Sorry but I cannot locate such object